amazon_msk/tests/common.py

```diff
@@ -65,3 +66,56 @@ def read_api_fixture():
 
 def read_e2e_api_fixture():
     return read_file(os.path.join(HERE, 'mock_boto3', 'boto3', 'list_nodes.json'))
+
+
+def assert_jmx_metrics(aggregator, tags, is_enabled=True):
+    expected_metrics = set()
+
+    for raw_metric_name, metric_name in JMX_METRICS_MAP.items():
+        if raw_metric_name.endswith('_total') and raw_metric_name not in JMX_METRICS_OVERRIDES:
+            expected_metrics.add('{}.count'.format(metric_name[:-6]))
+        else:
+            expected_metrics.add(metric_name)
+
+    expected_metrics.update(METRICS_FROM_LABELS)
+    expected_metrics.update(data['legacy_name'] for data in METRICS_WITH_NAME_AS_LABEL.values())
+    expected_metrics.update(data['legacy_name'] + ".count" for data in METRICS_WITH_NAME_AS_LABEL.values())
+    metrics_from_label_ids = [
```

Comment:
Small nit, could we move this to a constant like `METRICS_FROM_LABEL_IDS` outside of this test so future tests can use this same list?

Reply:
I'd wait for that to become a need and then perform the refactor. The current setup minimizes "jumping around", the variables are defined close to where they are used. This improves readability in my experience.